examples/experiments/decay/research_decay.ipynb

```diff
@@ -0,0 +1,508 @@
+{
```

Comment:
<p><em>Suggested change:</em></p><p> </p><p>The experiment without any decay shows the worst results (mean is the lowest, std is the highest). Therefore we can conclude that for a given dataset usage of learning rate scheduler makes sense.</p><p> </p><p>All tested schedulers achieve relatively equal results.&nbsp;Keep in mind, that they can be tuned more and used on other projects for better performance.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>